docs/src/docs/usage/false-positives.mdx

```diff
@@ -31,7 +31,7 @@ linters-settings:
 
 Exclude issue by text using command-line option `-e` or config option `issues.exclude`.
 It's helpful when you decided to ignore all issues of this type.
-Also, you can use `issues.exclude-rules` config option for per-path or per-linter configuration.
+Also, you can use `issues.exclude-rules` config option for per-path, per-linter, per-text or per-source configuration.
```

Comment:
```suggestion
Also, you can use `issues.exclude-rules` config option for per-path or per-linter configuration.
```
text and source are extra filters, they cannot be used standalone.

Reply:
oh okay, applied, thank you!